frontend/views/components/Tooltip.vue

```diff
@@ -177,10 +161,24 @@ export default {
         // That way the adjustPosition() method can have the same logic
         // applied in every tooltip as expected
         $this.adjustPosition()
+        window.addEventListener('resize', $this.adjustPosition)
+
+        if (bindings.value && bindings.value.manual) {
+          document.addEventListener('keydown', e => $this.trapFocus(e, el))
+          $this.lastFocus = document.activeElement
+          $this.focusEl(el)
+        }
       },
-      unbind (el) {
+      unbind (el, bindings, vnode) {
         if (el.parentNode) {
           el.parentNode.removeChild(el)
+          window.removeEventListener('resize', this.adjustPosition)
```

Comment:
The resize event should be removed always, even if there isn't a `parentNode`. Otherwise, we'll have a memory leak. And it should be `$this` instead of `this`.